runtime/onert/core/src/compiler/Fp32ToFp16Converter.cc

```diff
@@ -905,7 +905,7 @@ void Fp32ToFp16Converter::deleteContiguousOpSequences(
     VERBOSE(Fp32ToFp16Converter) << "Delete Node #" << first_node_ind.value() << std::endl;
 
     // Uses
-    for (auto &ind : first_node.getInputs().asUnique())
+    for (auto &ind : first_node.getInputs() | ir::Remove::duplicated | ir::Remove::optional)
```

Comment:
I'm not sure if applying `ir::Remove::optional` is okay. @YongseopKim Could you check this?

Reply:
@ᅟragmani Thank you again for review. The aim of this PR is to introduce `|` for filtering. I will make another PR to get review whether applying this filter is required or not. I will remove all `Remove::optional` filter in this PR.

Reply:
I see. And it looks good for me than `asUnique()`.